scripts/test/Abins/AbinsLoadPhonopyTest.py

```diff
@@ -31,10 +31,6 @@ def test_non_existing_file(self):
         with self.assertRaises(TypeError):
             _ = EuphonicLoader(input_ab_initio_filename=1)
 
-    @unittest.skipUnless(euphonic_available(), 'Optional dependency (euphonic) not available')
```

Comment:
Have we checked with Adam(?) this is okay to remove?

Reply:
Adam told me to